### ISSUE TITLE
Crash log

### ISSUE THREAD

**Issue by jrhaberstroh** [datamade/usaddress#199]
RepeatedLabelError: 
ERROR: Unable to tag this string because more than one area of the string has the same label

ORIGINAL STRING:  300 SE 1ST AVENUE 106
PARSED TOKENS:    [('300', 'AddressNumber'), ('SE', 'StreetNamePreDirectional'), ('1ST', 'StreetName'), ('AVENUE', 'StreetNamePreType'), ('106', 'StreetName')]
UNCERTAIN LABEL:  StreetName

**Comment by jrhaberstroh** [comment]
Not sure how to add BadParse label.

**Comment by jeancochrane** [comment]
Hey @jrhaberstroh,

Thanks for flagging this! I'm guessing this got raised during the `tag` method, yeah? If you've got more examples of this pattern that are failing, toss 'em here and we can train on them.

**Comment by jrhaberstroh** [comment]
Yep, these were on `tag`. I ran it on a larger training set using error handling, and I was able to get a handful of extra addresses. These are all from tax assessor's records. This is .05% of the full dataset, but I won't put more here unless that is of interest.

Warning: RepeatedLabelError raised for '613 & 613 2 W GRAHAM DR', leaving row blank
Warning: RepeatedLabelError raised for '2450 N ST NW', leaving row blank
Warning: RepeatedLabelError raised for '1307 E 2250 NORTH RD', leaving row blank
Warning: RepeatedLabelError raised for '33035 281ST ST; 33134 281ST ST', leaving row blank
Warning: RepeatedLabelError raised for '13851 NE 254 PRIVATE RD', leaving row blank
Warning: RepeatedLabelError raised for 'APT DE CORREOS # 914', leaving row blank
Warning: RepeatedLabelError raised for '539 W 1ST AVENUE 202', leaving row blank
Warning: RepeatedLabelError raised for '5363 I 55 S FRONTAGE RD', leaving row blank
Warning: RepeatedLabelError raised for '752 CS 2903 TUTTLE OK 73089', leaving row blank
Warning: RepeatedLabelError raised for '18781 NE 5TH TERRACE RD CITRA FL 321132296', leaving row blank
Warning: RepeatedLabelError raised for 'E 16295 ST HIGHWAY 82', leaving row blank
Warning: RepeatedLabelError raised for '6277 SEA HARBOR DR DEPT # DRIVE-', leaving row blank
Warning: RepeatedLabelError raised for 'C CORPORATE CENTER DR BOX # HILL', leaving row blank
Warning: RepeatedLabelError raised for '844 COACH WILLIAMS RD; 815 GRANGER RD', leaving row blank
Warning: RepeatedLabelError raised for '13660 NE 209TH TERRACE RD FORT MC COY FL 321345735', leaving row blank
Warning: RepeatedLabelError raised for '9988 ROUTE 403 HWY S', leaving row blank
Warning: RepeatedLabelError raised for '4920 W ST NW', leaving row blank
Warning: RepeatedLabelError raised for '568 CONTOUR; 8350 DENALI DR', leaving row blank
Warning: RepeatedLabelError raised for '9090 41ST ST NW LOT # 19', leaving row blank
Warning: RepeatedLabelError raised for '876 1/2 21 1/2 RD', leaving row blank
Warning: RepeatedLabelError raised for '506 6TH ST; 3525 WHITEHALL PARK DR STE 500', leaving row blank
Warning: RepeatedLabelError raised for '8415 8495TH ST 34', leaving row blank
Warning: RepeatedLabelError raised for '51 N ST NE', leaving row blank
Warning: RepeatedLabelError raised for '306 N 7TH ST; 216 E NORTH LOOP', leaving row blank
Warning: RepeatedLabelError raised for '406 2ND ST; 507 2ND ST', leaving row blank
Warning: RepeatedLabelError raised for '1533 100TH ST COUNTY RD H', leaving row blank
Warning: RepeatedLabelError raised for '1978 W LINCOLN RD; 3810 OLIVE ST', leaving row blank
Warning: RepeatedLabelError raised for '1035 GEORGE W ENGRAM BLVD', leaving row blank
Warning: RepeatedLabelError raised for '330 HOWARD ST TOWNSHIP OF WASHINGTON NJ 076765034', leaving row blank
Warning: RepeatedLabelError raised for '215 THIRD ST NORTH BCH ST AUGUSTINE FL 32084', leaving row blank
Warning: RepeatedLabelError raised for '169 SUNRISE HIGHWAY SERVICE RD', leaving row blank
Warning: RepeatedLabelError raised for 'MOBERLY ROAD LOT 90X100 PLU', leaving row blank
Warning: RepeatedLabelError raised for '3 3/4 MI N ON FM491', leaving row blank
Warning: RepeatedLabelError raised for '153 CR3155 (FROG POND GATE 5)', leaving row blank
Warning: RepeatedLabelError raised for 'NC 9 HWY', leaving row blank
Warning: RepeatedLabelError raised for '106 & 108 N MONTGOMERY ST', leaving row blank
Warning: RepeatedLabelError raised for 'N 900 W AT W 300 N', leaving row blank
Warning: RepeatedLabelError raised for 'GRAVEL RD E OF RD', leaving row blank
Warning: RepeatedLabelError raised for 'ST RD 39 COUNTY RD 750 S', leaving row blank
Warning: RepeatedLabelError raised for '1307 E 2250 NORTH RD', leaving row blank
Warning: RepeatedLabelError raised for '11 OCEAN ISLE WEST BLVD # 3-2', leaving row blank
Warning: RepeatedLabelError raised for '539 W 1ST AVENUE 202', leaving row blank
Warning: RepeatedLabelError raised for '18781 NE 5TH TERRACE RD', leaving row blank
Warning: RepeatedLabelError raised for '14 E POLK ST & 11/19/21 S WILLIAMS ST', leaving row blank
Warning: RepeatedLabelError raised for 'REYNOLDS AVE FAIRVIEW AVE', leaving row blank
Warning: RepeatedLabelError raised for '8014 56 NYS ROUTE 289', leaving row blank
Warning: RepeatedLabelError raised for '1100 BLK S TILLOTSON OPAS', leaving row blank
Warning: RepeatedLabelError raised for 'NEWTON RD COUNTY ROAD 70', leaving row blank
Warning: RepeatedLabelError raised for '13551 NE 209TH COURT RD', leaving row blank
Warning: RepeatedLabelError raised for 'LOT 17 THE RIDGE PHS #2 LOT 17', leaving row blank
Warning: RepeatedLabelError raised for 'BUFFALO CREEK ROAD STATE ROAD 16', leaving row blank
Warning: RepeatedLabelError raised for '1608 NE 2 1/2 AVE', leaving row blank
Warning: RepeatedLabelError raised for 'GA HIGHWAY 93 N AT GA HIGHWAY 111 N', leaving row blank
Warning: RepeatedLabelError raised for 'WS N 27TH 20 N MITCHELL', leaving row blank
Warning: RepeatedLabelError raised for 'LT 17 BLK 52', leaving row blank
Warning: RepeatedLabelError raised for '4270 W 1140 N SKYLINE DR', leaving row blank
Warning: RepeatedLabelError raised for '35251 263 OLD CO ROUTE 7', leaving row blank
Warning: RepeatedLabelError raised for 'BILL CURLEE ROAD STATE ROAD 1415', leaving row blank
Warning: RepeatedLabelError raised for '4275 LONG PINE CH ROAD STATE ROAD 122', leaving row blank
Warning: RepeatedLabelError raised for 'WARREN LOOP RD, PRIVATE RD N', leaving row blank
Warning: RepeatedLabelError raised for 'ROUTE TWO FIFTY', leaving row blank
Warning: RepeatedLabelError raised for '3742 MAIN ST RES-545 S MAIN ST', leaving row blank
Warning: RepeatedLabelError raised for 'PRIVATE ROAD NORTH OF CLIFTON RD', leaving row blank
Warning: RepeatedLabelError raised for '1035 GEORGE W ENGRAM BLVD', leaving row blank
Warning: RepeatedLabelError raised for 'HIGHWAY 19 AT HIGHWAY 172', leaving row blank
Warning: RepeatedLabelError raised for 'S 600 W AT W US HIGHWAY 52', leaving row blank
